File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -823,18 +710,16 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Signing an existing document
 
-You can load the signature field from the existing PDF document and add certificate to the document as follows.
+You can load the signature field from the existing PDF document and add certificate to the document by using [PdfLoadedSignatureField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html) class as follows.
```

Comment:
You can load the signature field from the existing PDF document using [PdfLoadedSignatureField]() class and add certificate to the document using [PdfCertificate]() class.

Reply:
changed the content